src/Uno.UI/UI/Xaml/Controls/SwipeControl/SwipeItems.h.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// Imported in uno on 2021/03/21 from commit 307bd99682cccaa128483036b764c0b7c862d666
+// https://github.com/microsoft/microsoft-ui-xaml/blob/307bd99682cccaa128483036b764c0b7c862d666/dev/SwipeControl/SwipeItem.h
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Windows.Foundation.Collections;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class SwipeItems
+	{
+		//public:
+		//SwipeItems();
+
+		#region IVector
+		//winrt.SwipeItem GetAt(uint index);
+		//uint Size();
+		//winrt.IVectorView<winrt.SwipeItem> GetView();
+		//bool IndexOf(winrt.SwipeItem & value, uint32_t& index);
+		//void SetAt(uint index, winrt.SwipeItem & value);
+		//void InsertAt(uint index, winrt.SwipeItem & value);
+		//void RemoveAt(uint index);
+		//void Append(winrt.SwipeItem & value);
+		//void RemoveAtEnd();
+		//void Clear();
+
+		// TODO:
```

Comment:
I don't think any of these weird pseudo-Linq methods are actually part of the .NET projection, you can just remove them.

```csharp
#region Assembly Windows.Foundation.UniversalApiContract, Version=8.0.0.0, Culture=neutral, PublicKeyToken=null, ContentType=WindowsRuntime
// C:\Program Files (x86)\Windows Kits\10\References\10.0.18362.0\Windows.Foundation.UniversalApiContract\8.0.0.0\Windows.Foundation.UniversalApiContract.winmd
#endregion

using System.Collections.Generic;
using Windows.Foundation;
using Windows.Foundation.Metadata;

namespace Windows.UI.Xaml.Controls
{
	//
	// Summary:
	//     Represents a collection of SwipeItem objects.
	[Composable(typeof(ISwipeItemsFactory), CompositionType.Public, 327680, "Windows.Foundation.UniversalApiContract")]
	[ContractVersion(typeof(UniversalApiContract), 327680)]
	[MarshalingBehavior(MarshalingType.Agile)]
	[Static(typeof(ISwipeItemsStatics), 327680, "Windows.Foundation.UniversalApiContract")]
	[Threading(ThreadingModel.Both)]
	[WebHostHidden]
	public class SwipeItems : DependencyObject, ISwipeItems, IList<SwipeItem>, IEnumerable<SwipeItem>
	{
		//
		// Summary:
		//     Initializes a new instance of the SwipeItems class.
		public SwipeItems();

		//
		// Summary:
		//     Gets or sets a value that indicates the effect of a swipe interaction.
		//
		// Returns:
		//     A value of the enumeration that indicates the effect of a swipe interaction.
		//     The default is **Reveal**.
		public SwipeMode Mode { get; set; }
		//
		// Summary:
		//     Identifies the Mode dependency property.
		//
		// Returns:
		//     The identifier for the Mode dependency property.
		public static DependencyProperty ModeProperty { get; }
	}
```

Reply:
Actually they are part of the public API ... https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.swipeitems.first?view=winrt-19041#Windows_UI_Xaml_Controls_SwipeItems_First

Reply:
They sure are, if you're using C++. That's why I specified 'the .NET projection' ;)

I was going to copy-paste the actual info from the binary but I see I already did, so... see above...